src/main/java/com/exasol/errorreporting/ExaError.java

```diff
@@ -0,0 +1,24 @@
+package com.exasol.errorreporting;
+
+/**
+ * Facade for building Exasol error messages.
+ */
+public class ExaError {
+
+    private ExaError() {
+        // empty on purpose
+    }
+
+    /**
+     * Get a builder for error messages.
+     * <p>
+     * These builder calls can be crawled using https://github.com/exasol/error-code-crawler-maven-plugin.
```

Comment:
I wonder if we really want to include a reference to the crawler in our source code. It's just a text and not a direct dependency, but anyway. Maybe we should remove this line.

Reply:
They will be coupled anyway.If we change this API we also need to change the crawler. So I think the link helps new team members to understand this connection.

Reply:
The crawler is couple to the error reporting, but not the other way around.